examples/echo-server.rs

```diff
@@ -2,11 +2,11 @@
 //!
 //! You can test this out by running:
 //!
-//!     cargo run --example server 127.0.0.1:12345
+//!     cargo run --features="async-std-runtime" --example echo-server 127.0.0.1:12345
```

Comment:
Would you like this command updates split into another PR because they don't have anything to do with the "not broadcast close" thing? I guess I was being a little lazy and just stuffing everything together.

Reply:
That's ok with me as long as it's a separate commit in the same PR :)